README.md

```diff
@@ -9,7 +9,8 @@ The default stack in this repo includes three modular components:
 |----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | ML Code                    | Example ML project structure, with unit tested Python modules and notebooks using [MLflow pipelines](https://mlflow.org/docs/latest/pipelines.html)                | Quickly iterate on ML problems, without worrying about refactoring your code into tested modules for productionization later on.                                                        |
 | ML Resource Config as Code | ML pipeline resources (training and batch inference jobs, etc) defined through [Terraform](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/terraform/) | Govern, audit, and deploy changes to your ML resources (e.g. "use a larger instance type for automated model retraining") through pull requests, rather than adhoc changes made via UI. |
-| CI/CD                      | [GitHub Actions](https://github.com/actions) workflows to test and deploy ML code and resources                                                                    | Ship ML code faster and with confidence: ensure all production changes are performed through automation and that only tested code is deployed to prod                                   |
+| CI/CD                      | One of the following: <ul><li>[GitHub Actions](https://github.com/actions) workflows to test and deploy ML code and resources</li><li>[Azure DevOps](https://azure.microsoft.com/en-gb/products/devops/) to test and deploy ML code and resources</ul>              | Ship ML code faster and with confidence: ensure all production changes are performed through automation and that only tested code is deployed to prod                                   |
```

Comment:
Super minor nit, maybe here we could just say "GitHub Actions or Azure DevOps workflows to test and deploy ML code and resources ...", instead of switching to a bulleted list, just to save a bit of space/format a bit more nicely

Reply:
yea agreed on this actually. fixed